_layouts/category.html

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+---
+<h2>Posts related to: {{ page.title }}</h2>
+
+{% unless page.content == '' %}
+  <p>{{ page.content }}</p>
+{% endunless %}
+
+{% for post in site.categories[page.category] %}
+<div class="post-preview">
+    <a href="{{ post.url | prepend: site.baseurl }}">
+        <h2 class="post-title">            
+            {{ post.title }}
+        </h2>
+        {% if post.subtitle %}
+        <h3 class="post-subtitle">
+            {{ post.subtitle }}
+        </h3>
+        {% endif %}
+    </a>
+    <p class="post-meta">
+      Posted by 
+      {% if post.author %}
+        {{ post.author }}
+      {% else %}
+        {{ site.title }}
+      {% endif %} 
+      on {{ post.date | date: "%B %d, %Y" }}
```

Comment:
In https://github.com/evafroehlich/website/blob/3011c16092f1c934106f23ba9f986a549f3d18c4/_layouts/post.html#L6 wird `date_to_string` verwendet. Das scheint die Funktion zu sein, die seiten-weite Einstellung zum Dateformat berücksichtigt.